recipes/julia/julia-wrapper.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+JULIA_HOME="$(dirname "${BASH_SOURCE[0]}")"
+
+# Set JULIA_PKGDIR to $PREFIX/share/julia/site to avoid contaminating
+# user's ~/.julia/.  This script will modify Pkg.dir() and
+# Base.LOAD_CACHE_PATH[1] appropriately.  Note that wrapper script is
+# used rather than /etc/julia/juliarc.jl because LOAD_CACHE_PATH is
```

Comment:
you should also be able to just replace its contents

Reply:
So I missed this before and maybe you did as well @tkelman, but the comment here says that "`LOAD_CACHE_PATH` is configured in base/sysimg.jl which is loaded before juliarc.jl." Hence its manipulation in the wrapper script. Knowing very little about Julia's loading procedure I can't comment further, but it sounds like changes to `LOAD_CACHE_PATH` in `juliarc` will simply be ignored. Thoughts on this?

Reply:
the sysimg.jl value is just a default, it's a mutable array so can be modified anywhere

Reply:
Right. That much is clear. What is less clear is what has already been done based on `LOAD_CACHE_PATH` at this point. Are there any docs on Julia's loading procedure?

Reply:
FWIW opened issue ( https://github.com/conda-forge/julia-feedstock/issues/10 ) to get rid of the wrapper script and @dfornika's PR ( https://github.com/conda-forge/julia-feedstock/pull/11 ) is an attempt to solve this. Maybe you can advise on how best to proceed.